cmd/set/accessRestriction/accessRestriction.go

```diff
@@ -0,0 +1,97 @@
+package accessRestriction
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/loginradius/lr-cli/api"
+	"github.com/loginradius/lr-cli/cmdutil"
+	"github.com/spf13/cobra"
+)
+
+
+
+type domain struct {
+	BlacklistDomain    string `json:"blacklistdomain"`
+	WhitelistDomain    string `json:"whitelistdomain"`
+	DomainMod string `json:"domainmod"`
+}
+
+
+func NewaccessRestrictionCmd() *cobra.Command {
+	opts := &domain{}
+
+	cmd := &cobra.Command{
+		Use:   "access-restriction",
+		Short: "Updates whitelisted/blacklisted Domains/Emails",
+		Long:  `Use this command to update the whitelisted/blacklisted Domains/Emails.`,
+		Example: heredoc.Doc(`
+		$ lr set access-restriction --blacklist-domain <old-domain> --new-domain <new-domain>
+		Blacklist Domains/Emails have been updated successfully
```

Comment:
We can use `--domain` instead of `--blacklist-domain`

Reply:
@mohammed786  Actually in set command we can set either blacklist domain or whitelist domain thats why it is --blacklist-domain/--whitelist-domain